src/components/UI/FormSelectorFilterable.vue

```diff
@@ -61,13 +61,17 @@ const id = randomString()
     @include background-primary;
   }
   border-radius: 4px;
-  height: 30px;
-  :global(.vs__dropdown-toggle),
+  // height: 35px;
```

Comment:
改めてになりますが、元の高さはそのままにしておいていただけると…！
```suggestion
  height: 30px;
```